docs/ref/contrib/gis/geos.txt

```diff
@@ -167,6 +167,21 @@ Properties
 
 Returns the coordinates of the geometry as a tuple.
 
+.. attribute:: GEOSGeometry.dims
+
+Returns the dimension of the geometry:
+
+* returns ``0`` for :class:`Point` and :class:`MultiPoint`
+
+* returns ``1`` for :class:`LineString` and :class:`MultiLineString`
+
+* returns ``2`` for :class:`Polygon` and :class:`MultiPolygon`
+
+* returns ``-1`` for empty :class:`GeometryCollection`
+
+* for non-empty :class:`GeometryCollection` returns maximum dimension of
```

Comment:
I think we can omit the "returns" on each item in the list and say "The maximum dimension of its elements for non-empty :class:`GeometryCollection`\s." for the last one. 

Also, you may omit a blank line between list items.